api/utils/blobaccess/maven/access_test.go

```diff
@@ -6,7 +6,6 @@ import (
 	. "github.com/mandelsoft/goutils/testutils"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
```

Comment:
The line break should stay. The linebreak after that import can be removed IIRC

Reply:
I can for sure, revert that, but that change was made by: `make format`... and I do not plan to change that now ;-)